tests/dialects/test_op_builder.py

```diff
@@ -1,3 +1,5 @@
+import pytest
```

Comment:
Oh, I haven't catched that last time, but we should move this file to `tests/test_op_builder.py`